rosdep/base.yaml

```diff
@@ -795,6 +795,11 @@ emacs:
   gentoo: [virtual/emacs]
   openembedded: [emacs@meta-oe]
   ubuntu: [emacs]
+embree:
+  arch: [embree]
+  debian: [libembree-dev]
+  fedora: [embree]
```

Comment:
This is only available from 32, while we also support 28 (melodic), 23 and 24 (kinetic).
I see in ubuntu we sometimes specify in which distro versions a package is available, but IDK if that's necessary in fedora.
Maybe @nuclearsandwich knows (?)

Reply:
Fedora releases themselves are only supported for ~13 months I don't think we need to block on past releases here.

Reply:
Will you support fedora 32 in the future?
If you will do so, I will stay these lines, if not, I will remove these lines.

Reply:
I think keeping it is fine

Reply:
> Will you support fedora 32 in the future?

Noetic targets fedora 32 btw